lib/tools.js

```diff
@@ -104,6 +105,7 @@ function asString (str) {
 }
 
 function asJson (obj, msg, num, time) {
+  asJsonChan.start.publish({ instance: this, arguments })
```

Comment:
Could you add a test verifying that the "active span" (or just dummy object in AsyncLocalStorage) is exactly the same as when the logging function is called?